build.sbt

```diff
@@ -1,31 +1,62 @@
-crossScalaVersions := Seq("2.13.6", "2.12.14")
-scalaVersion := crossScalaVersions.value.head
+import xerial.sbt.Sonatype._
 
-name := "library-example"
+ThisBuild / scalaVersion := "3.0.1-RC2"
 
-// also used as a `groupId` by Sonatype
-organization := "ch.epfl.scala"
+lazy val library = project
+  .in(file("library"))
+  .settings(
+    name := "library-example",
+    organization := "ch.epfl.scala", // also used as a `groupId` by Sonatype
+    description := "A library that does nothing useful",
+    crossScalaVersions := Seq("3.0.0", "2.13.6", "2.12.14"),
+    scalaVersion := crossScalaVersions.value.head,
+    libraryDependencies += "com.github.scalaprops" %% "scalaprops" % "0.8.3" % Test,
+    testFrameworks += new TestFramework("scalaprops.ScalapropsFramework")
+  ).settings(publishSettings)
 
-libraryDependencies += "com.github.scalaprops" %% "scalaprops" % "0.8.3" % Test
-testFrameworks += new TestFramework("scalaprops.ScalapropsFramework")
+lazy val publishSettings = Def.settings(
+  sonatypeProjectHosting := Some(GitHubHosting("scalacenter", "library-example", "julien.richard-foy@epfl.ch")),
+  // indicate the open source licenses that apply to our project
+  licenses := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
+  // publish to the Sonatype repository
+  publishTo := sonatypePublishToBundle.value,
+  // binary compatibility check
+  mimaPreviousArtifacts := Set.empty // Disabled on `master` branch
+)
 
-description := "A library that does nothing useful"
+lazy val documentation = project
+  .in(file("site"))
+  .settings(siteSettings)
+  .enablePlugins(MdocPlugin)
+  .dependsOn(library)
```

Comment:
We should configure this project to not be published (e.g. with `publish / skip := true`)